frontend/src/utils/configs.js

```diff
@@ -1,4 +1,2 @@
-export const USER_SVC_BASE_URL =
-  window.__RUNTIME_CONFIG__.REACT_APP_URI_USER_SVC || "http://localhost:8000";
-export const MATCH_SVC_BASE_URL =
-  window.__RUNTIME_CONFIG__.REACT_APP_URI_MATCH_SVC || "http://localhost:8001";
+export const API_URL =
+  window.__RUNTIME_CONFIG__.REACT_APP_API_URL || "http://localhost";
```

Comment:
I think this may cause problems for local development as we need the ports to differentiate the services.

Suggest to create separate config files for PROD and DEV environments since we know NODE_ENV will be prod when the frontend is deployed on GCP.